server/backend/mlx_engine/model_kit/model_kit.py

```
@@ -0,0 +1,236 @@
+                mx.array(prompt_tokens),
+                self.cache_wrapper,
+                generate_args,
+                self.draft_model,
+                speculative_decoding_toggle,
+                prompt_progress_callback,
+            ), None
+        ### WITH IMAGES PROMPT PROCESSING ###s
+        if self.vision_add_on is None:
+            raise ValueError(
+                "Vision add-on is not loaded, but images were provided for processing"
+            )
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)